HTML-CSS/Tasarım ödevi/index.html

```diff
@@ -0,0 +1,23 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>Tasarım</title>
+</head>
+<body>
+    <div class="grid">
+        <div id="box-1"></div>
+        <div id="box-2"></div>
+        <div id="box-3"></div>
+        <div id="box-4"></div>
+        <div id="box-5"></div>
+        <div id="box-6"></div>
+        <div id="box-7"></div>
+        <div id="box-8"></div>
+        <div id="box-9"></div>
```

Comment:
Genelde ID, CSS içerisinde kullanılmaz. Bunun sebebi sahip olduğu spesifikasyon farklılıklarıdır. Elementlerini stillendirirken class'larla stillendirmeni tavsiye ederim. 

Bu konuda detaylı bilgiye şu makaleden ulaşabilirsin: https://dev.to/clairecodes/reasons-not-to-use-ids-in-css-4ni4